application/forms/RotationConfigForm.php

```diff
@@ -1416,6 +1416,8 @@ private function yieldRecurrenceRules(int $count): Generator
                 $fromDay = (int) $options['from_day'];
                 $toDay = (int) $options['to_day'];
                 $interval = (int) $options['interval'];
+                $fromAt = new DateTime($options['from_at']);
+                $toAt = new DateTime($options['to_at']);
```

Comment:
In light of the current development regarding timezones, I'd love to see a solution that doesn't involve datetime objects, unless you really need them. As of right now, I think numerical comparisons might suffice.